packages/teams-js/src/public/interfaces.ts

```diff
@@ -749,6 +749,44 @@ export interface DeepLinkParameters {
   subEntityWebUrl?: string;
 }
 
+/**
+ * @hidden
+ * Shared Dialog Properties
+ */
+export interface BaseDialogInfo {
+  /*
+   * The requested size of the dialog
+   */
+  size: DialogSize;
+
+  /**
+   * Title of the task module.
+   */
+  title?: string;
+}
+
+/**
+ * Data structure to describe dialog information needed to open a adaptive card based dialog.
```

Comment:
nit: **an** adaptive **card-based** dialog

Reply:
Made changes in the latest commit